src/Compilers/CSharp/Portable/Errors/ErrorCode.cs

```diff
@@ -2251,6 +2251,7 @@ internal enum ErrorCode
         ERR_CollectionBuilderAttributeInvalidMethodName = 9186,
         ERR_CollectionBuilderAttributeMethodNotFound = 9187,
         ERR_CollectionBuilderNoElementType = 9188,
+        ERR_InlineArrayForEachNotSupported = 9189,
```

Comment:
Hm, I will have to renumber all my error codes again if you merge first https://github.com/dotnet/roslyn/pull/69230/commits/9ba1fee40d532910893f12f0e3e0b1e7c5cbf8ca#diff-3ac66c25866171b5ebdd5ff2622127096c9b78b7f724d7936b64872faa977894